pkg/util/strategicpatch/patch.go

```diff
@@ -963,13 +972,170 @@ func HasConflicts(left, right interface{}) (bool, error) {
 	}
 }
 
+// MergingMapsHaveConflicts returns true if the left and right JSON interface
```

Comment:
This comment looks almost identical to https://github.com/kubernetes/kubernetes/blob/master/pkg/util/strategicpatch/patch.go#L930.  What's the difference?